### PR TITLE
Border disappears in pseudobutton's disabled state

### PR DESCRIPTION
**Pull request by gela-d**
/cc @mishanga

### CODE REVIEW

**Comment by gela-d** [comment]
@mishanga Mishaaaangaaa :)

**Comment by mishanga** [comment]
21_podborka_31

File: design/common.blocks/button/_theme/button_theme_normal.roo [hunk 1]
```diff
@@ -178,7 +178,7 @@
         color: rgba(0, 0, 0, 0.3);
         background: rgba(0, 0, 0, 0.08);
 
-        &:before
+        &:not(.button_pseudo):before
```
Comment (mishanga) [inline design/common.blocks/button/_theme/button_theme_normal.roo]: This is wrong way. You should just remove `important` from here:
https://github.com/bem/bem-components/blob/126deac621f3174566d63530a9d827b97ac21086/design/common.blocks/button/_theme/button_theme_normal.roo#L132

**Comment by gela-d** [comment]
@mishanga mimimi :)
Done.

**Pull request merged**